<?xml version="1"?>
<launch>
    <param name="robot_description" command= "$(find xacro)/xacro --inorder '$(find panda_description)/Panda/panda.urdf'"/>
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
        <param name="use_gui" value="False"/>
    </node>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d /home/starslab_go/stars_lab_ws/panda_robot/src/panda_description/rviz/panda.rviz" output="screen"/>
</launch>
    

